### PR TITLE
Sandbox: Update to latest near membrane version ^0.13.0

### PR DESCRIPTION
**Pull request by academo**
**What is this feature?**

Updates the near-membrane library used in the frontend sandbox

Note: This new version removes the instrumentation previously available in the library so I'm removing the existing code. The existing code was not really doing much to improve errors

**Why do we need this feature?**

To keep up to date with features and security fixes

**Who is this feature for?**

All plugins running inside the frontend sandbox

**Which issue(s) does this PR fix?**:

Superseeds this PR https://github.com/grafana/grafana/pull/74376

**Special notes for your reviewer:**

Please check that:

- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -259,9 +259,9 @@
     "@lezer/common": "1.0.2",
     "@lezer/highlight": "1.1.3",
     "@lezer/lr": "1.3.3",
-    "@locker/near-membrane-dom": "^0.12.15",
-    "@locker/near-membrane-shared": "^0.12.15",
-    "@locker/near-membrane-shared-dom": "^0.12.15",
+    "@locker/near-membrane-dom": "0.13.3",
```
Comment (jackw) [inline package.json:262]: Following on the conversation from the renovate PR... you can add it to the renovate ignore list [here](https://github.com/grafana/grafana/blob/main/.github/renovate.json5#L6-L19).

File: public/app/features/plugins/sandbox/sandbox_plugin_loader.ts [hunk 2]
```diff
@@ -130,31 +130,14 @@ async function doImportPluginModuleInSandbox(meta: PluginMeta): Promise<System.M
...
     });
 
     // fetch plugin's code
     let pluginCode = '';
     try {
       pluginCode = await getPluginCode(meta);
     } catch (e) {
-      throw new Error(`Could not load plugin ${meta.id}: ` + e);
-      reject(new Error(`Could not load plugin ${meta.id}: ` + e));
+      reject(new Error(`Could not load plugin code ${meta.id}: ` + e));
```
Comment (academo) [inline public/app/features/plugins/sandbox/sandbox_plugin_loader.ts:140]: Intentional change. The existing code was not rejecting the promise and potentially keep the wait for a plugin load forever.

File: package.json (hunk 1 above)
Comment (academo) [reply package.json:262]: I added it to the list as part of this PR

**Review by sunker — APPROVED** [review]
Don't have that much context, but looks good to me!

**Review by leventebalogh — APPROVED** [review]
LGTM 👍 

What's gonna be the difference (generally) between how non-caught errors from plugins inside the sandbox looked like before this change, and once we merge this?

**Comment by academo** [comment]
> LGTM 👍
> 
> What's gonna be the difference (generally) between how non-caught errors from plugins inside the sandbox looked like before this change, and once we merge this?

It only changes where the error is thrown. It used to show up inside the sandbox loader file now it'll show in the plugin loader file (which is where the whole transaction starts).

Also this prevents this promise to hang in the air for ever and create a never-ending loading situation.

**Pull request merged**